src/applications/ivc-champva/10-10D/containers/IntroductionPage.jsx

```diff
@@ -25,34 +24,19 @@ const IntroductionPage = props => {
       />
 
       <p>
-        If you’re the spouse or child of a Veteran with disabilities or a
-        Veteran who has died, you may be able to get health insurance through
-        the Civilian Health and Medical Program of the Department of Veterans
-        Affairs (CHAMPVA). Apply online now.
+        If you’re the spouse, dependent, or survivor of a Veteran or service
```

Comment:
I think this should be "dependant".  See [this](https://www.easybib.com/guides/grammar-guides/vocabulary/confusing-words/dependant-vs-dependent/#:~:text=Dependant%20is%20a%20noun%20that,an%20adjective%20describing%20a%20status.).